test/functional/helpers.lua

```diff
@@ -410,6 +410,10 @@ end
 --    clear('-e')
 --    clear{args={'-e'}, args_rm={'-i'}, env={TERM=term}}
 function module.clear(...)
+  -- Close the old sesion before starting a new one.
+  s = module.get_session()
```

Comment:
is there a reason local is not used? and why didn't our linter catch this?

```suggestion
  local s = module.get_session()
```

Reply:
No reason other than lack of sleep. Time for a cup of coffee I guess :)
Thank you for the review!